apache/netbeans
# [NETBEANS-2172] & [NETBEANS-2398] Fix for web.xml errors

*What this pull request does*
Fixes the errors when creating a web.xml file.  Before this fix, the parser will warn about the <web-app> tag not being recognized and XML validation will fail due to the .xsd files not being known

*Root Cause Analysis*
It seems that the JavaEE Deployment Descriptor .xsd files used to be a part of the JavaEE Deployment Descriptor loader module, but are now part of the J2EE Deployment Descriptor API module.

Possibly they moved / removed & re-added during the licencing of JavaEE for Apache?  I'm assuming that their new location is the correct/intended one.

The Netbeans XML catalog "Enterprise Deployment Descriptor Catalog" was still pointing to the .xsd files being part of the JEE DD Loader module, so various things were falling over when trying to access them

* Files Changed*
* /enterprise/j2ee.ddloaders/src/org/netbeans/modules/j2ee/ddloaders/catalog/EnterpriseCatalog.java
Resource Path constant changed from nbres:/org/netbeans/modules/j2ee/ddloaders/catalog/resources/ to nbres:/org/netbeans/modules/j2ee/dd/impl/resources/

*Testing*
* Make sure Java EE plugin activated
* Tools-> DTD & XML Schemas -> Expand Enterprise Deployment Descriptors catalog
* Chose any .xsd and click Open in Editor
Before this PR: An error will be thrown "Could not connect to URL nbres:/org/netbeans/modules/j2ee/ddloaders/catalog/resources/web-app_3_1.xsd. No such resource was found."
After this PR: .xsd file will be opened in Netbeans Editor
* Create a new Java Web Project, add a new web.xml file
Before this PR: File will be highlighted with errors / validate XML will fail
After this PR: No errors
